api-server/server/routes/remoteApis.js

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+'use strict'
+
+const LOGGER = require("../logger").logger
+const express = require("express")
+const openapiSampler = require('openapi-sampler');
+const refParser = require('json-schema-ref-parser');
+const { api, event, connection } = require("@varkes/app-connector")
+module.exports = {
+    router: router
+}
+
+function getAll(req, res) {
+    LOGGER.debug("Getting all APIs")
+    var err = assureConnected()
+    if (err) {
+        res.status(400).send({ error: err })
+    } else {
+        api.findAll().then((result) => {
+            res.status(result.statusCode).send(result.body);

```

Comment:
the api object should not return statusCode and so on, instead it should either succeed or fail with error message.

Think of error handling!

Reply:
Reimplemented as we discussed